src/Employer/Employer.Web/Controllers/VacancyManageController.cs

```diff
@@ -28,7 +27,7 @@ public VacancyManageController(VacancyManageOrchestrator orchestrator, IHostingE
         }
 
         [HttpGet("manage", Name = RouteNames.VacancyManage_Get)]
-        public async Task<IActionResult> ManageVacancy(VacancyRouteModel vrm)
+        public async Task<IActionResult> ManageVacancy(VacancyRouteModel vrm, string applicationUserName,string applicationReviewStatus,bool setApplicationStatus= false)
```

Comment:
Can we not pass the application id here and fetch related values instead of trusting query string?